src/main/java/com/questhelper/helpers/mischelpers/herbrun/HerbRun.java

```diff
@@ -175,8 +173,7 @@ public void setupConditions()
 		accessToHarmony = new QuestRequirement(QuestHelperQuest.THE_GREAT_BRAIN_ROBBERY, QuestState.FINISHED);
 		accessToWeiss = new QuestRequirement(QuestHelperQuest.MAKING_FRIENDS_WITH_MY_ARM, QuestState.FINISHED);
 		accessToTrollStronghold = new QuestRequirement(QuestHelperQuest.MY_ARMS_BIG_ADVENTURE, QuestState.FINISHED);
-		accessToFarmingGuildPatch = new Conditions(new FavourRequirement(Favour.HOSIDIUS, 60),
-			new SkillRequirement(Skill.FARMING, 65));
+		accessToFarmingGuildPatch = new Conditions(new SkillRequirement(Skill.FARMING, 65));
```

Comment:
This could be simplified to just the `SkillRequirement`, omitting the `Conditions`

Reply:
Good suggestion, went ahead and added that change as well.